plugins/opencritic.py

```diff
@@ -2,26 +2,54 @@
 import datetime
 import re
 import fuzzywuzzy
-from fuzzywuzzy import process
 
+from fuzzywuzzy import process
 from cloudbot import hook
 
+url = 'https://api.opencritic.com/api/game/'
+headers = {
+    'content-type': "application/json",
+    'cache-control': "no-cache"
+}
+platformList = {
+    ('switch', 'nintendo', 'nintendo switch'): '32',
+    ('sony', 'playstation', 'playstation 4', 'ps4', 'psn'): '6',
+    ('microsoft', 'ms', 'xbox', 'xbox one', 'xbox 1', 'xb1', 'xbl'): '7',
+    ('windows', 'pc'): '27',
+    ('3ds', 'nintendo 3ds'): '36',
+    ('vive', 'htc', 'htc vive'): '30',
+    ('rift', 'oculus', 'oculus rift'): '29',
+    ('vr', 'hmd'): '29,30',
+    ('vita', 'ps vita', 'psp'): '33'
+}
+
+
+def platformCategory(text):
+    platformId = ''
+
+    for platform, id in platformList.items():
+        if text.lower() in platform:
+            platformId = id
+
+    if len(platformId) == 0:
+        platformId += '6,7,27,29,30,32,33,36'
+
+    return platformId
```

Comment:
completely untested, but this should be more efficient.

```suggestion
    def platform_category(text):  # try not to use camel case
        platform_id = '6,7,27,29,30,32,33,36'
        text = text.lower()
        for platform, p_id in platformList.items():
            if text in platform:
                return id
        return platform_id
```